src/store/files.js

```diff
@@ -88,6 +103,16 @@ const actions = {
 	appendFiles(context, files = []) {
 		context.commit('updateFiles', files)
 	},
+
+	/**
+	 * Set list of all .nomedia/.noimage files
+	 *
+	 * @param {object} context the store mutations
+	 * @param {Array} paths list of files
+	 */
+	setNomediaPaths(context, paths) {
+		context.commit('setNomediaPaths', paths)
```

Comment:
Could you debug this in the console?
```suggestion
		logger.debug('Ignored paths', { paths })
```

Reply:
Like this?

```js
    setNomediaPaths(context, paths) {
		console.debug('Ignored paths', { paths })
		context.commit('setNomediaPaths', paths)
	},
```